packit/api.py

```diff
@@ -481,7 +485,24 @@ def watch_copr_build(
             build = client.build_proxy.get(build_id)
             if build.state == state_reported:
                 continue
-            gh_state, description = COPR2GITHUB_STATE[build.state]
+            logger.debug(f"COPR build {build_id}, state = {build.state}")
+            try:
+                gh_state, description = COPR2GITHUB_STATE[build.state]
+            except KeyError as exc:
+                logger.error(f"COPR gave us an invalid state: {exc}")
+                gh_state, description = "error", "Something went wrong."
+            if build.state == "succeeded":
+                msg = (
+                    "Congratulations! The build(s) finished successfully.\n"
+                    "If you want to test your changes, please follow these steps:\n\n"
+                    f"1. Open this URL: {repo_url}\n",
+                    "2. Download .repo file for the target you're interested "
+                    "in and place it to `/etc/yum.repos.d/`.\n",
+                    "3. Install the packages either with dnf if you're on Fedora,"
+                    " or yum if your system is RHEL.\n"
+                    "\nPlease note that the RPMs should be used only in a testing environment.",
+                )
+                self.upstream_local_project.git_project.pr_comment(pr_id, msg)
```

Comment:
We might want to do the reporting in job handle, for case this function is triggered by CLI.